src/Store.js

```diff
@@ -78,6 +79,37 @@ class Store {
     return transaction(() => data.map((obj) => this.addModel(type, obj)))
   }
 
+  /**
+   * Saves a collection of records via a bulk-supported JSONApi
+   * endpoint. All records need to be of the same type.
+   *
+   * @method bulkSave
+   * @param {String} type
+   * @param {Array} records
+   */
+  bulkSave = (type, records, options = {}) => {
```

Comment:
Can we give `records` a default of `records = []`?

Also, how should we handle the case of `records` being empty? Should we even make a request?

Reply:
good call 👍